### PR TITLE
Add support for Vivaldi Browser

### PR DESCRIPTION
**Pull request by ttuygun**
Added Vivaldi Browser support related with the issue #1225.

### CODE REVIEW

File: mackup/applications/vivaldi.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = Vivaldi
+
+[configuration_files]
+Library/Application Support/Vivaldi
+
```
Comment (lra) [inline mackup/applications/vivaldi.cfg]: Can you remove the extra newline?

File: mackup/applications/vivaldi.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = Vivaldi
+
+[configuration_files]
+Library/Application Support/Vivaldi
```
Comment (lra) [inline mackup/applications/vivaldi.cfg]: Has it been tested? Isn't there a lot of cache in here?

Comment (ttuygun) [reply mackup/applications/vivaldi.cfg]: > Has it been tested? Isn't there a lot of cache in here?

Actually, caches are located in Library/Caches/Vivaldi

**Comment by ttuygun** [comment]
https://github.com/lra/mackup/pull/1479

**Pull request closed without merging**